matrix/common/etl/transformers/cell_expression.py

```diff
@@ -165,6 +172,75 @@ def _parse_10x_bundle(self, bundle_dir):
 
         return cell_lines, expression_lines
 
+    def _parse_optimus_bundle(self, bundle_dir):
+        """
+        Parses optimus analysis files into PSV rows for cell and expression Redshift tables.
+        """
+        keys = self._parse_keys(bundle_dir)
+        cell_suspension_id = json.load(open(
+            os.path.join(bundle_dir, "cell_suspension_0.json")))['provenance']['document_id']
+
+        # read zarr expression matrix
+        store = DCPZarrStore(bundle_dir=bundle_dir)
+        root = zarr.group(store=store)
+        expr_values = root.expression_matrix.expression[:]
```

Comment:
I think this load the whole dense count array into memory. Are you sure that's necessary? I think it could get pretty big for some bundles.

Reply:
Oh good call! Updated to process zarr in chunks.